bin/scripts/config-helper.js

```diff
@@ -13,199 +13,200 @@
   See the License for the specific language governing permissions and
   limitations under the License.
 */
-const fs = require('fs');
-const figlet = require("figlet");
-const { prompt } = require('enquirer');
+import fs from 'fs';
+import figlet from 'figlet';
+import { prompt } from 'enquirer';
```

Comment:
This change either needs to be reverted or we need to make the necessary changes to maintain functionality. This is run from a cli as a script so we can't use the import pattern unless we rename this file to `.mjs`, update the `config` script in `package.json` to use the `.mjs` extension and also update these import statements to:
```
import fs from 'fs';
import figlet from 'figlet';
import enquirer from 'enquirer';

const prompt = enquirer.prompt;
```

`enquirer` is a CommonJS module.